dotCMS/src/main/java/com/dotcms/rest/api/v1/user/UserResource.java

```diff
@@ -440,19 +448,26 @@ private void revertLoginAsSessionInfo(final HttpServletRequest request, final Ho
 	private void setImpersonatedUserSite(final HttpServletRequest req, final String userID) throws DotDataException, DotSecurityException {
 		final HttpSession session = req.getSession();
 		final User user = this.userAPI.loadUserById(userID);
+		Logger.info(this,"USER: " + user);
 		final String currentSiteID = (String) session.getAttribute(com.dotmarketing.util.WebKeys.CMS_SELECTED_HOST_ID);
+		Logger.info(this,"currentSiteID: " + currentSiteID);
 		Host currentSite;
 		try {
 			currentSite = this.siteAPI.find(currentSiteID, user, false);
 		} catch (final DotSecurityException e) {
+			Logger.info(this,"currentSite: " + "DOTSECURITY");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "currentSite: " appears 5 times in this file; the first occurrence is on line 458](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5280129)